### DISCUSSION TITLE
Individual License

### DISCUSSION THREAD

**Discussion by clt-pereira** [datalust/seq-tickets#2332] [Help] [answered]
Good afternoon everyone!
Using a free individual license, I would like to know if I can use my application in production normally, that is, from what I understand I can install it on a dedicated server and access the dashboard normally, since this is done by just 1 user. Does that check?

**Comment by nblumhardt** [comment, marked as answer]
Hi Cleiton, thanks for your message.

There's no distinction made in Seq licensing between production and non-production use, so the Individual license can be used in production environments.

The other terms of the Individual license still apply, including non-redistribution and non-transferabilty; e.g. the server must be yours/not a customer's, the person who installs Seq must be the one using it, etc. - https://datalust.co/doc/eula-current.pdf has the full set of requirements.

Hope this helps,
Nick

**Comment by clt-pereira** [comment]
Good morning everyone!

I understand, however, I still have questions about what would be "server installer" mentioned in the Team licensing and not mentioned in the Individual license, for example.
Regarding the installation on a personal server and not on the client's server, this type of blocking is conditioned in some way during the installation using the individual license, that is, I provide software in the SaaS model and it is hosted on my own server. company on Windows, can I provide this software to customers and still use the Individual license to use Seq?

**Reply by nblumhardt** [reply]
Seq licensing doesn't limit who you can provide your own software to - just Seq itself. So running a SaaS product of your own for customers, and monitoring it with Seq behind the scenes, is fine under the Individual license as it stands today, subject to the other conditions imposed by the EULA.

Hosting Seq for customers to use would be a different scenario, and would be restricted by the license. The EULA text is the best source of detailed information on all this.

**Reply by clt-pereira** [reply]
There was only one point missing, as I didn't find this information anywhere, what would be the "**server installer**" that is not included in the Individual license?
I would like to know the difference in operation between the individual license without the "server installer" and the Team license that has the server-installer." I need this information so that the company can evaluate the licensing purchase.

**Reply by nblumhardt** [reply]
Hi! "Server installs" refers to the number of (shared) servers the license can be used on. This doesn't apply to the Individual license; because it's primarily used on personal workstations and we like to use quite a few ourselves, there's currently no restriction on these installs.
